create-register.sh

```diff
@@ -0,0 +1,263 @@
+#!/bin/sh
+
+
+
+set -x
+set -eu
+
+
+
+usage() {
+    echo "Usage: $0 <register-name> <environment> <optional-data-file-location> <optional-data-file-type>"
+    echo ""
+    echo "Launches a register in given environment with the provided name"
+    echo "You must have setup credential repository on your machine before running the script"
+    echo "You must run this script from deployment repository"
+}
+
+
+if [ "$#" -lt 2 ]; then
+    echo "Wrong number of arguments"
+    usage; exit
+fi
+
+
+REGISTER_NAME=$1
+ENV=$2
+BUCKET_NAME=openregister.$ENV.config
+INDEXER_CONFIG_LOCATION=/tmp/indexer.properties
+CURRENT_WORKING_DIR=`pwd`
+
+REGISTER_DB_NAME=${REGISTER_NAME//-/_}
+CONFIG_FILE_DIRECTORY=${CURRENT_WORKING_DIR}/register-configurations
+
+function cdIntoCredentailsRepository(){
+  pushd ~/.registers-pass/
+  export PASSWORD_STORE_DIR=$(cd $(dirname -- $BASH_SOURCE) && pwd)
```

Comment:
this won't work because this bash file isn't in the password store dir, it's in the `deployment` dir...